src/stream/src/executor/dispatch.rs

```diff
@@ -162,6 +164,11 @@ impl DispatchExecutorInner {
     async fn dispatch(&mut self, msg: Message) -> Result<()> {
         match msg {
             Message::Chunk(chunk) => {
+                self.metrics
+                    .actor_out_record_cnt
+                    .with_label_values(&[&self.actor_id.to_string()])
```

Comment:
We don't need to multiply output number.

> I might be captious and have somehow high standard upon the PR instead of dealing with it as a kind of research code. Just a reminder, if you're demanding on the progress of your project, you can always fork and develop on your own branch instead of getting everything carefully reviewed and merged to main, which might be time-consuming for you. Anyway, feel free to ping me and other developers involving in the scaling thing and get things reviewed. For me, a large PR might take ~3days to a week to review, depending on my workload at that time. Small PRs like this can be reviewed much more quickly.

The true rate metrics and input/output number are metrics used for auto-scaling. I think I will add more metrics after this PR. I will contact with other developers for auto-scaling.